bin/local-env/install-wordpress.sh

```diff
@@ -33,82 +30,96 @@ echo ''
 # dirty up the tests.
 if [ "$1" == '--reset-site' ]; then
 	echo -e $(status_message "Resetting test database...")
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI db reset --yes --quiet
+	wp db reset --yes --quiet
 fi
 
 if [[ ! -z "$WP_VERSION" ]]; then
 	# Potentially downgrade WordPress
 	echo -e $(status_message "Downloading WordPress version $WP_VERSION...")
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI core download --version=${WP_VERSION} --force --quiet
+	wp core download --version=${WP_VERSION} --force --quiet
 fi
 
 # Install WordPress.
 echo -e $(status_message "Installing WordPress...")
-# The `-u 33` flag tells Docker to run the command as a particular user and
```

Comment:
Sounds like this is no longer necessary / applicable?

Reply:
It is, although in the `wp` function we use the username `xfs` instead which is what user 33 refers to. This is consistent with the `user` set in the `docker-compose.yml` so it's easier to reason about.

Reply:
Makes sense!